eng/Version.Details.xml

```diff
@@ -19,8 +19,7 @@
     </Dependency>
     <Dependency Name="VS.Redist.Common.NetCore.SharedFramework.x64.6.0" Version="6.0.0-rtm.21501.9" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>e013f14c5330375a18656dea8dca14e5602a3ae0</Sha>
-      <SourceBuildTarball RepoName="runtime" ManagedOnly="true" />
+      <Sha>6c5c32d56ec74280783664c2526937d23cfc3467</Sha>
```

Comment:
@sfoslund, what is the purpose of this "Revert versions changes" commit?  Reverting the SourceBuild related changes is going to be impactful.

Reply:
We usually let automatic insertions update these files and try to avoid touching them in merge PRs. Which lines need to stay? Only the `<SourceBuildTarball...` or are there version updates you need as well?

Reply:
I think every change is needed except this commit-sha update for the runtime.

Reply:
It looks like there's still something wrong with tarball creation, can you take another look?

Reply:
I pushed a commit which resolves the source-build patch conflicts.